src/auth/choicewindow.h

```diff
@@ -0,0 +1,43 @@
+#ifndef CHOICEWINDOW_H
+#define CHOICEWINDOW_H
+
+#include <QDialog>
+
+QT_BEGIN_NAMESPACE
+class QLabel;
+class QListWidget;
+class QAbstractButton;
+class QListWidgetItem;
+class QDialogButtonBox;
+QT_END_NAMESPACE
+
+class ChoiceWindow : public QDialog
+{
+    Q_OBJECT
+public:
+    ChoiceWindow(QStringList &usersFilesList, QWidget *pParent = nullptr);
+
+private:
+    QListWidget *mpUsersWidgetList;
+
+    QPushButton *mpUnnamedUserButton;
+    QPushButton *mpNewUserButton;
+
+    QDialogButtonBox *mpButtonBox;
+
+    // design parameters
+    int mBasicStretch               = 1;
```

Comment:
` int mBasicStretch               = 1;`
                                                                                                          What is the purpose of leaving so large space?

Reply:
it is ok